src/App.js

```diff
@@ -1,9 +1,48 @@
-import React from 'react';
+import React from "react";
+import './App.css';
+
+import TypoGraphy from '@material-ui/core/Typography'
+
+import {Login} from "./Components/Login/Login";
+import {Register} from "./Components/Register/Register";
+import {BrowserRouter as Router, Route, Switch, Redirect} from "react-router-dom";
+import Dashboard from "./Components/Dashboard/Dashboard";
+import Header from "./Components/Header/Header";
+import Footer from "./Components/Footer/Footer";
+
 
 function App() {
+    const auth = localStorage.getItem('AUTH');
+
     return (
-        <div>
-            This homework requires a Hero!
+        <div className='app-wrapper'>
+
+
+            <div className='app-wrapper-content'>
+
+
+
+            <Router>
+                {!auth && <Switch>
+                    {/*<Route path="/login" component={Login}/>*/}
+                    <Route path='/login'
+                           render={() => <Login padding='5' margin='1'/>}
+                    />
+                    <Route path="/dashboards" component={Dashboard}/>
+
+                    <Redirect to="/login"/>
+                </Switch>}
+                {!!auth && <Switch>
+                    <Route path="/register" component={Register}/>
+                    <Route exact path="/" component={Login}/>
+                    <Redirect to="/"/>
+                </Switch>}
+            </Router>
+
+            </div>
+
+            <Footer/>
+            <Header/>
         </div>
     );
 }
```

Comment:
I think it might be smth like this:

```
            <Router>
                {!auth && <Switch>
                    <Route path='/login'
                           render={() => <Login padding='5' margin='1'/>}
                    />
                    <Route path="/register" component={Register}/>
                    <Redirect to="/login"/>
                </Switch>}
                {!!auth && <Switch>
                    <Route path="/" component={Dashboard}/>
                    <Redirect to="/"/>
                </Switch>}
            </Router>
```
but up to you

Reply:
fix